_sass/components/_glossary.scss

```diff
@@ -65,6 +65,7 @@
     width: 50px;
     height: 50px;
     background-color: transparent;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.